src/go/intelligence/int_test/test_wrapper.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/sh
+##H Script for automation of testing process of the intelligence module.
+##H Usage: test_wrapper.sh <work_directory>
+##H
+
+# If user provided the input parameter
+if [ $# -ne 1 ]; then
+    perl -ne '/^##H/ && do { s/^##H ?//; print }' <$0
```

Comment:
please replace with
```
cat $0 | grep "^##H" | sed -e "s,##H,,g"
```
as you add additional dependency on perl.

Reply:
fixed this.